Signal/src/Models/TSMessageAdapaters/TSAnimatedAdapter.m

```diff
@@ -40,16 +39,17 @@ - (instancetype)initWithAttachment:(TSAttachmentStream *)attachment incoming:(BO
         _cachedImageView = nil;
         _attachment      = attachment;
         _attachmentId    = attachment.uniqueId;
-        _image           = [attachment image];
-        _fileData        = [NSData dataWithContentsOfURL:[attachment mediaURL]];
```

Comment:
Not only were we retaining the loaded image, we were also retaining its contents in memory.